workflow/applications/gefs.py

```diff
@@ -59,6 +59,8 @@ def get_task_names(self):
         if self.nens > 0:
             tasks += ['efcs']
 
+        tasks += ['repairf0306']
```

Comment:
I think this task should be added to the workflow only for replay cases. This task should probably be conditioned. I suggest reviewing the extractvars task to see how a task can be made optional in the workflow.